src/storage/mod.rs

```diff
@@ -384,6 +640,41 @@ mod backend_tests {
         Ok(())
     }
 
+    fn test_get_range(storage: &Storage) -> Result<(), Error> {
+        let blob = Blob {
+            path: "foo/bar.txt".into(),
+            mime: "text/plain".into(),
+            date_updated: Utc::now(),
+            compression: None,
+            content: b"test content\n".to_vec(),
+        };
+
+        storage.store_blobs(vec![blob.clone()])?;
+
+        assert_eq!(
+            blob.content[0..=4],
+            storage
+                .get_range("foo/bar.txt", std::usize::MAX, (0..=4).into(), None)?
+                .content
+        );
+        assert_eq!(
+            blob.content[5..=12],
+            storage
+                .get_range("foo/bar.txt", std::usize::MAX, (5..=12).into(), None)?
+                .content
+        );
+
+        for path in &["bar.txt", "baz.txt", "foo/baz.txt"] {
```

Comment:
I'm not sure what `baz.txt` is testing, but it definitely seems wrong that we're not testing the original file name.
```suggestion
        for path in &["bar.txt", "baz.txt", "foo/bar.txt"] {
```

Reply:
this is adapted from `test_get_object`, and is testing for the error that the path is not found. 

So using the filename from above here would break the test.